runtimes/parachain/src/lib.rs

```diff
@@ -492,7 +478,7 @@ parameter_types! {
 	pub const LaunchPeriod: BlockNumber = 7 * DAYS;
 	pub const VotingPeriod: BlockNumber = 7 * DAYS;
 	pub const FastTrackVotingPeriod: BlockNumber =  3 * HOURS;
-	pub const MinimumDeposit: Balance = 1 * DOLLARS;
+	pub const MinimumDeposit: Balance = 2 * DOLLARS;
```

Comment:
nice fix 😂

Reply:
Yeah... I was not sure but since this will be changed before the launch I thought it would be fine 😅